### ISSUE TITLE
Changing an Icon Does Not Work

### ISSUE THREAD

**Issue by FrenchToucan** [Goooler/LawnchairRelease#52]
### Describe the bug

Introduced in 240201, attempting to change the icon of any app on the home screen or in the drawer results in a warning that "Lawnchair keeps stopping" error window and then the icon does not change. The crash occurs after the new icon is selected.

**Comment by Goooler** [comment]
```
java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
    at androidx.appcompat.app.AppCompatDelegateImpl.androidx.appcompat.app.AppCompatDelegateImpl.createSubDecor(Unknown Source:755)
                                                    ensureSubDecor
    at androidx.appcompat.app.AppCompatActivity.androidx.appcompat.app.AppCompatDelegateImpl.onPostCreate(Unknown Source:9)
                                                onPostCreate
    at android.app.Instrumentation.callActivityOnPostCreate(Instrumentation.java:1530)
    at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3950)
    at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:225)
    at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:205)
    at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:177)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:98)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2547)
    at android.os.Handler.dispatchMessage(Handler.java:111)
    at android.os.Looper.loopOnce(Looper.java:242)
    at android.os.Looper.loop(Looper.java:362)
    at android.app.ActivityThread.main(ActivityThread.java:8393)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:992)
```

**Comment by Goooler** [comment]
Check https://github.com/Goooler/LawnchairRelease/releases/tag/14.0.0-20240203

**Comment by FrenchToucan** [comment]
Fixed in 20240203. Thanks!
